docs/README.md

```diff
@@ -1,15 +1,141 @@
+# Crystal Book
 
-Crystal is a programming language with the following goals:
+Welcome to the documentation for the Crystal language.
 
-* Have a syntax similar to Ruby (but compatibility with it is not a goal).
-* Be statically type-checked, but without having to specify the type of variables or method parameters.
-* Be able to call C code by writing bindings to it in Crystal.
-* Have compile-time evaluation and generation of code, to avoid boilerplate code.
-* Compile to efficient native code.
+Crystal is a language for humans and computers. These materials help humans understand the language and its ecosystem.
 
-From here, you can jump to anywhere you want in this document. Although, if you are new to the Crystal Language, we suggest that you continue reading the [Getting started](./getting_started/README.md) section.
+## Learning Materials
 
-**Crystal's standard library is documented in the [API docs](https://crystal-lang.org/api).**
+These instructions and courses help you got get to know the language and how to use it.
+
+<div class="cards">
+  <div class="card">
+    <h3><a href="getting_started/">
+      Getting started
+    </a></h3>
+    <p>Install Crystal and get it running.</p>
+  </div>
+  <div class="card">
+    <h3>
+      Tutorials
+    </h3>
+    <p>Introductory courses for beginners and advanced learners.</p>
+    <ul>
+      <li><a href="tutorials/basics/index.html">
+        Basics
+      </a></li>
+    </ul>
+  </div>
+  <div class="card">
+    <h3><a href="guides/">
+      Guides
+    </a></h3>
+    <p>Detailed examples for practical applications.</p>
+  </div>
+  <div class="card">
+    <h3>
+      Tools
+    </h3>
+    <ul>
+      <li><a href="https://github.com/crystal-lang/crystal/wiki/Online-playgrounds">
+        Online Playgrounds
+      </a></li>
+      <li><a href="https://github.com/veelenga/awesome-crystal#editor-plugins">
+        IDE Integrations
+      </a></li>
+      <li><a href="guides/ci/index.html">
+        CI Integrations
+      </a></li>
+    </ul>
+  </div>
+</div>
+
+---
+
+## Grow in Crystal
+
+The core documentation of the Crystal language, standard library and tooling.
+
+<div class="cards">
+  <div class="card">
+    <h3><a href="syntax-and-semantics/">
+        Language Reference
+    </a></h3>
+    <p>Specification of the language.</p>
+  </div>
+  <div class="card">
+    <h3><a href="https://crystal-lang.org/api">
+        Standard Library API
+    </a></h3>
+    <p>Documentation of the standard library.</p>
+  </div>
+  <div class="card">
+    <h3>
+      Shards
+    </h3>
+    <p>Discover the ecosystem of Crystal libraries.</p>
+    <ul>
+      <li><a href="https://crystal-lang.org/community/#shards">
+        Discovering Shards
+      </a></li>
+      <li><a href="https://github.com/crystal-lang/shards/blob/master/docs/shard.yml.adoc">
+        Specification
+      </li>
+      <li><a href="the_shards_command/README.md">
```

Comment:
At least on the preview site this links ends in a 404. (NB: the CI is not complaining about it 🤔 )

Reply:
Yeah, it needs `.html` extension. It seems mkdocs only recognizes (and warns) about links in markdown format and not in HTML. But I don't think we can use Markdown links inside HTML and we need HTML for the layout.

Reply:
@oprypin Any ideas?

Reply:
Sent https://github.com/straight-shoota/crystal-book/pull/1

Reply:
Oh, nice 🥇